crates/evm/src/trace/node.rs

```diff
@@ -110,19 +109,19 @@ impl CallTraceNode {
 
         if let RawOrDecodedCall::Raw(ref bytes) = self.trace.data {
             let inputs = if bytes.len() >= SELECTOR_LEN {
-                if self.trace.address == CHEATCODE_ADDRESS.to_ethers() {
+                if self.trace.address == CHEATCODE_ADDRESS {
                     // Try to decode cheatcode inputs in a more custom way
                     utils::decode_cheatcode_inputs(func, bytes, errors, verbosity).unwrap_or_else(
                         || {
-                            func.decode_input(&bytes[SELECTOR_LEN..])
+                            func.abi_decode_input(&bytes[SELECTOR_LEN..], false)
```

Comment:
@DaniPopes should this be true?

Reply:
`false` has the same behavior as `ethabi`. Doesn't really matter here